notification/docs/HowToRun.md

```diff
@@ -99,13 +99,15 @@ Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION
+| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------- |
+| `adyenPaymentMethodsToNames` | Key-value object where key is `paymentMethod` returned in the notification and value is the custom localized name that will be saved in CTP `payment.paymentMethodInfo.method`.                              | `{scheme: {en: 'Credit Card'}, pp: {en: 'PayPal'}, klarna: {en: 'Klarna'}, gpay: {en: 'Google Pay'}}` |
+| `port`                       | The port number on which the application will run.                                                                                                                                                           | 443                                                                                                   |
+| `logLevel`                   | The log level (`trace`, `debug`, `info`, `warn`, `error`, `fatal`).                                                                                                                                          | `info`                                                                                                |
+| `keepAliveTimeout`           | Milliseconds to keep a socket alive after the last response ([Node.js docs](https://nodejs.org/dist/latest/docs/api/http.html#http_server_keepalivetimeout)).                                                | Node.js default (5 seconds)                                                                           |
+| `removeSensitiveData`        | Boolean attribute. When set to "false", Adyen fields with additional information about the payment will be saved in the interface interaction and in the custom fields.                                      | true                                                                                                  |
+| `notificationBaseUrl`        | Publicly available URL of the Notification module. In case of any payment changes, Adyen will call this URL and pass the notification in body. This attribute is used when calling `npm run setup-resources` |                                                                                                       |
+| `apiKey`                     | This key will be used to make authenticated API requests to Adyen. This attribute is used when calling `npm run setup-resources`                                                                             |                                                                                                       |
```

Comment:
Description just says where and how this `apiKey` is used, but It would be good if it explains what it is.

Reply:
OK, what would you write here?

Reply:
I am not sure what is `apiKey` here. 
Ex: for 'notificationBaseUrl' we wrote,  `Publicly available URL of the Notification module`.
Same way a short sentence which explains about this key(where can someone get or generate this from).

Reply:
It is actually `A String that is used to make authenticated API requests to Adyen` which is more or less what is there.

Reply:
Okay anyhow i think it is also okay to leave it like this then.